wbooks/src/app/screens/bookList/index.tsx

```diff
@@ -3,9 +3,16 @@ import { FlatList, ListRenderItem, SafeAreaView, View } from 'react-native';
 import { BOOKS_MOCK } from '@constants/mockBooks';
 import Book from '@components/book';
 import { Book as IBook } from '@interfaces/bookInterface';
+import { connect } from 'react-redux';
 
 import styles from './styles';
 
+const mapStateToProps = (state: any) => {
```

Comment:
you can create an interface of State, create the file reduxInterfaces.ts

Reply:
Any was changed to Interface